t/op/infnan.t

```diff
@@ -576,4 +576,20 @@ cmp_ok('-1e-9999', '==', 0,     "underflow to 0 (runtime) from neg");
     is($x, ' Inf ', "string shall be ' Inf ' after UV/NV conversion");
 }
 
+# "-Inf" should be converted to IV consistently
+{
+    # Enclosed in a string eval so that majority of infnan.t are run
+    # before loading integer.pm, just in case.
+    eval <<'EOC';
```

Comment:
Just in case of what? If there's a risk of `use integer` having buggy effects outside its scope, that's not really the responsibility of this test file to cope with. But I see no reason to believe there is such a risk - and certainly we don't use such caution in other much more fundamental test files.

(Also if there were good reason to keep the eval, you should at the very least check for it dying. But far better to remove it.)